data/json/furniture.json

```diff
@@ -3134,7 +3157,7 @@
     "type": "furniture",
     "id": "f_displaycase",
     "name": "display case",
-    "description": "Display your stuff.  Securely.",
+    "description": "Display your stuff securely.  Features fancy velvet couches cocked at a fetching angle.",
```

Comment:
Is there actual couches inside the case? I think this is a poor description, especially for translation issues.

Reply:
Agreed. Shouldn't add things to the description that require new disassembly items.